#!/usr/bin/env python3

'''
again, look at the traceback and fix!
'''


def function(x=5):
    print('function({}) called!'.format(x))

func = 'function'

func(9)
